docs/deep-dives/data/preenrollment_bias.md

```diff
@@ -0,0 +1,116 @@
+---
+
+# Automatically Countering Preenrollment Bias
+
+TL;DR: Nimbus has the capability to adjust metrics to account for preenrollment bias/randomization failures and to improve the precision of inferences, when possible. Currently this is configured by default for guardrails (averages only) but can also be used for custom analyses. This is expected to reduce the frequency of false positives, of which we believe many were caused by randomization failure.
+
+## Preenrollment Bias
+
+In order to generate evidence for a causal hypothesis, we must guarantee that all sources of [confounding](https://en.wikipedia.org/wiki/Confounding) are accounted for. We can either do this by manually controlling for all confounders (which is quite difficult), or we can use randomized experiments in which the randomization process guarantees that, _on average_, units in each treatment branch are balanced on all confounders.
+
+Randomization provides a guarantee of balance on average and across large numbers of experiments, but in practice, for any given experiment and confounding dimension there is the possibility of imbalance. This imbalance (or rather, confounding) presents a challenge to our goal of gathering causal evidence. An imbalance observed during the treatment period is indistinguishable from a treatment effect. So how can we detect this?
+
+## Retrospective A/A tests
+
+User behavior tends to be consistent over time. We've found week-to-week correlations of up to 80% for our key guardrail metrics. Given this strong correlation, we can look for evidence of imbalance during the _pre-experiment period_. We've been using the term preenrollment bias, though [others](https://www.statsig.com/blog/pre-experiment-bias-detection-statsig) use pre-experiment bias.
+
+During the pre-experiment period, our experimental cohorts should have no statistically significant difference across the dimensions (metrics) of interest. If there are statistically significant differences during the pre-experiment period, this is evidence of bias. This technique is called a [Retrospective A/A test](https://www.microsoft.com/en-us/research/articles/patterns-of-trustworthy-experimentation-pre-experiment-stage/).
+
+We now automatically run Retrospective A/A tests for all Nimbus experiments to test for imbalance in guardrail metrics. These can be found alongside the other statistical results in the [Jetstream data products](https://docs.telemetry.mozilla.org/datasets/jetstream.html#statistics-tables).
+
+In short, you can find them in the `moz-fx-data-experiments.mozanalysis.statistics_<slug>_preenrollment_<period>_1` where `<slug>` is the (snake case) experiment slug or ID (can be found using the Experimenter UI). We run analyses over 2 periods: the week prior to enrollment (`<period>` = `week`) and the 28-day period prior (`<period>` = `days_28`).
+
+For example:
+
+```sql
+SELECT *
+FROM `moz-fx-data-experiments.mozanalysis.statistics_<slug>_preenrollment_week_1`
```

Comment:
Since it's an example, maybe we should put a fake slug in there?

```suggestion
FROM `moz-fx-data-experiments.mozanalysis.statistics_fake_experiment_slug_preenrollment_week_1`
```